README.md

```diff
@@ -464,6 +464,9 @@ and run a SHA256() hash and compare the source and target results. Since each ro
 be returned in the result set, it is recommended to utilize the `--use-random-row` feature
 to validate a subset of the table.
 
+Please note that SHA256 is not a supported function on teradata systems. If you wish to perform
```

Comment:
Let's also add this disclaimer in the top level Row Validation docs here: https://github.com/GoogleCloudPlatform/professional-services-data-validator#row-validations